packages/grid_client/src/modules/nodes.ts

```diff
@@ -79,6 +81,20 @@ class Nodes {
   async setNodePower(options: NodePowerModel) {
     return (await this.client.nodes.setPower(options)).apply();
   }
+
+  @expose
+  @validateInput
+  @checkBalance
```

Comment:
we don't need to `validateInput` decorator for a method that doesn't have input.

also, for `checkBalance` user doesn't have to have tokens to do a query